hypervisor/arch/x86/cpu_primary.S

```diff
@@ -225,14 +225,14 @@ cpu_primary32_gdt_ptr:
     .align  0x1000
     .global cpu_boot32_page_tables_start
 cpu_boot32_page_tables_start:
-    /* 0x3 = (IA32E_COMM_P_BIT | IA32E_COMM_RW_BIT) */
+    /* 0x3 = (PAGE_PRESENT | PAGE_RW) */
```

Comment:
This is not to remove the unused MACRO definition...  It mismatch with the commit message.
BTW, why we need to change the name here? Can you explain a bit?

Reply:
This is wants to set the hypervisor MMU page table attributes not guest EPT page table attributes.
It should use the Macro definition for hypervisor MMU page table attributes.